### ISSUE TITLE
Negative exclusion filter doesn't work for some process names

### ISSUE THREAD

**Issue by ero-qt** [smourier/TraceSpy#19]
I'm using a negative exclusion filter on process names to only include a specific process. The filter is set up like this:
```
Filter On: Process Name
Filter Type: Regular Expression
Definition: ^((?!ProcessName).)*$
```

I have noticed that this filter works correctly for some processes (only traces from that process are shown), but does not work for others (no traces whatsoever are shown). The process names, of course, match perfectly. Turning off the filter does show traces from that process again.

**Comment by ero-qt** [comment]
Sorry, this appears to be fixed in `4.2.1.0`. I was still on `4.2.0.0`.

**Comment by roman-orekhov** [comment]
I'm not able to make it work with any of the processes. The logs are always shown
